client/offchain/src/lib.rs

```diff
@@ -139,6 +140,8 @@ impl<Client, Storage, Block> OffchainWorkers<
 			});
 			futures::future::Either::Left(runner.process())
 		} else {
+			let help = "Consider turning off offchain workers if they are not part of your runtime.";
```

Comment:
Problem is that validators need the offchain api, otherwise they don't send the heartbeat transactions. So, turning it off is not the best idea?